opus/application/static_media/js/browse.js

```diff
@@ -691,111 +725,298 @@ var o_browse = {
-                    o_browse.gallerySliderStep = galleryBoundingRect.x;
-                }
-                $("#op-observation-slider").slider({
-                    "value": obsNum,
-                    "step": o_browse.gallerySliderStep,
-                    "max": maxSliderVal,
-                });
-            }
-            $(`${tab} .op-slider-nav`).removeClass("op-button-disabled");
-            // update startobs in url when scrolling
-            o_hash.updateHash(true);
+            // When we update the slider, the following steps are run:
+            // 1. Get the current startObs (top left item in gallery view in current page)
+            //    and top obsNum (top item in table view and one of top row items in gallery
+            //    view) that scrollbar is at.
+            //    - If browser is resized, realign doms by deleting some cached obSnum to make
+            //      sure first cached obs are at the correct row boundary. If we don't do so,
+            //      all rendered obs will be with the wrong row boundary after browser resizing.
+            // 2. Get the max value for slider.
+            // 3. Get the scrollbar offset for both gallery and table view.
+            // 4. Update the slider value and data in infiniteScroll instances and URL using
+            //    the data obtained from above 2 steps.
+            let obsNumObj = o_browse.realignDOMAndGetStartObsAndScrollbarObsNum(selector, browserResized);
+            let obsNum = obsNumObj.startObs;
+            let currentScrollObsNum = obsNumObj.scrollbarObsNum;
+            let maxSliderVal = o_browse.getSliderMaxValue();
+            let scrollbarOffset = o_browse.getScrollbarOffset(obsNum, currentScrollObsNum);
+            o_browse.setSliderParameters(obsNum, currentScrollObsNum, scrollbarOffset.galleryOffset,
+                                       scrollbarOffset.tableOffset);
         }
 
+        let currentSliderValue = $(`${tab} #op-observation-slider`).slider("option", "value");
+        let currentSliderMax = $(`${tab} #op-observation-slider`).slider("option", "max");
+
         let galleryImages = o_browse.countGalleryImages();
-        if (opus.prefs[startObsLabel] + numThumbnails < galleryImages.x * galleryImages.y) {
+        if ((opus.prefs[startObsLabel] + numObservations - 1) < galleryImages.x * galleryImages.y ||
+            (currentSliderValue <= 1 && currentSliderValue >= currentSliderMax)) {
             // disable the slider because the observations don't fill the browser window
-            // $("#op-observation-slider").slider({
+            // $(`${tab} #op-observation-slider`).slider({
             //     "value": 1,
             //     "step": o_browse.gallerySliderStep,
             //     "max": 1,
             // });
             $(`${tab} .op-slider-pointer`).css("width", "3ch");
+            // Make sure slider always move the the most left before being disabled.
+            $(`${tab} .op-observation-number`).html("1");
```

Comment:
Does setting the HTML to "1" actually move the slider? I thought you would have to change it's "value" property.